tsfc/driver.py

```diff
@@ -291,30 +297,54 @@ def prepare_coefficient(integral_type, coefficient, name):
         return funarg, [], expression
 
     # Interior facet integral + mixed / vector element
-    name_ = name + "_"
-    shape = (2, fiat_element.space_dimension())
```

Comment:
Maybe a short comment on what this packing is doing?  And why there are two options?